neonavigation_launch/launch/demo.launch

```diff
@@ -1,8 +1,11 @@
 <?xml version="1.0"?>
 <launch>
+  <arg name="use_path_with_velocity" default="true" />
```

Comment:
For visualizing the `path_velocity` in Rviz by default, updating readme is better to clone https://github.com/at-wat/neonavigation_rviz_plugins/ .